### DISCUSSION TITLE
How to exclude words or inline block from **automatic** translation?

### DISCUSSION THREAD

**Discussion by ebouchut** [squidfunk/mkdocs-material#5569] [Q&A] [answered]
In order to prevent _Google Translate_ (and others, like _Yandex_ for instance) from **automatically** translating a word/group of words or whatever inline block like `span`, I need a way to **add the `notranslate` CSS class**.

- ℹ️ To prevent a **single** word from being translated, I surround it with back-ticks like so: ``` `word` ```.
- ℹ️ To prevent a **paragraph** (a block) from being translated, I add `{ .notranslate }` on the very next line after the paragraph.
- ❓However when this is not a block/paragraph but an inline block like when there are **several words** in a sentence, or even a single word that is not code, I have not found a solution. Hence my question.

Would you have something in mind or in your tool belt to do this?

**Comment by ebouchut** [comment, marked as answer]
An option would be to use regular HTML (see below), but I would prefer wrapping this into a Markdown using an extension of some sort if possible.
```html
... <span class="notranslate">these words won't be automatically translated</span> ...
```

**Reply by squidfunk** [reply]
Regular HTML is the way to go, as you would need the `span` element anyways. However, there might be a Markdown extension that allows to wrap inline elements in `span` elements by using a specific syntax, but it would definitely be non-standard. Not sure if it's a good idea to use that, since `span` elements are much more compatible with other Markdown renderers (e.g. GitHub).

**Comment by ebouchut** [comment]
👍 Thank you for the advice @squidfunk, will stick to using `span` elements then.
